File: csharp2/Handler.cs
using System;
using Amazon.Lambda.Core;
using Amazon.Lambda.APIGatewayEvents;

[assembly:LambdaSerializer(typeof(Amazon.Lambda.Serialization.Json.JsonSerializer))]

namespace AwsDotnetCsharp
{
    public class Handler
    {
       public APIGatewayProxyResponse Hello(APIGatewayProxyRequest apigProxyEvent, ILambdaContext context)
       {
           return new APIGatewayProxyResponse {
             StatusCode = 200,
             Body = "hello"
           };
       }
    }
}

File: aws/aws-fsharp2.0/Handler.fs
namespace AwsDotnetFsharp

open Amazon.Lambda.Core

[<assembly:LambdaSerializer(typeof<Amazon.Lambda.Serialization.Json.JsonSerializer>)>]
do ()
    
module Handler =
    open Amazon.Lambda.APIGatewayEvents
 
    let hello(_:APIGatewayProxyRequest, _:ILambdaContext): APIGatewayProxyResponse =
        let response = APIGatewayProxyResponse()
        response.StatusCode <- 200
        response.Body <- "hello"
        response